scripts/wait-for-quorum-network.js

```diff
@@ -24,12 +24,13 @@ async function retry(fn, retries = 5, interval = 1000) {
 }
 
 const web3 = new Web3();
+const interval = 5000;
 web3.setProvider(
   `http://${developmentNetworkConfig.host}:${developmentNetworkConfig.port}`
 );
 
 console.log("Waiting for the quorum network to be ready...");
 
-retry(web3.eth.getAccounts, (interval = 5000))
+retry(web3.eth.getAccounts, interval)
```

Comment:
This change just seems neutral to me.

Reply:
The retry function defines `interval` as the 3rd positinal argument and `async function retry(fn, retries = 5, interval = 1000)` and this invocation mistakenly results in 5000 retries at 1 sec intervals! whoa!

Reply:
Oh, yikes, so there's a problem here that needs fixing then? (In the original code, I mean, rather than in the PR.)